ray-operator/test/support/test.go

```diff
@@ -121,3 +126,34 @@ func (t *T) NewTestNamespace(options ...Option[*corev1.Namespace]) *corev1.Names
 	})
 	return namespace
 }
+
+func (t *T) StreamKubeRayOperatorLogs() {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
This should probably use `t.Ctx()` instead, which is already cancellable.

Reply:
Hi @astefanutti,

Actually, I do notice that I could use `t.Ctx()` instead and I also tried it. It was canceled correctly, but the log
```go
t.T().Logf("Stop tailing logs from the pod %s/%s: %v", pod.Namespace, pod.Name, scanner.Err())
```
didn't print out. I guess that was just because the `t.Ctx()` was canceled too late, however, the missing log already made me less confident the approach.